Statistics module in Process home

htmlboy opened the pull request:
#### :tophat: What? Why?
Redesigned highlighted statistics module in home. 

#### :pushpin: Related Issues
- Related to #5635 

#### :clipboard: Subtasks
- [x] Add `CHANGELOG` entry
- [ ] Add documentation regarding the feature 
- [ ] Add/modify seeds
- [ ] Add tests
- [ ] Another subtask

### :camera: Screenshots (optional)
![Screenshot 2020-02-10 at 18 22 05](https://user-images.githubusercontent.com/468685/74173637-c0815a80-4c32-11ea-9489-28f22dfd6e0f.png)

carolromero commented:
Hi @htmlboy, good job! Nothing to do with the initial mockup, thankfully :joy: 

Just one thing: I just did a test to see how it works in context and maybe it would work better to line it up on the left like the other elements? What do you think?

![imatge](https://user-images.githubusercontent.com/23509895/74147026-acbe0000-4c02-11ea-8bf1-ff94d76f387d.png)

htmlboy commented:
@carolromero I thought this module was aimed to the home page, should we have it in the process page too?

carolromero commented:
@htmlboy oops nope, this issue specifically refers to changes in the homepage of a Participary Process :grimacing:
Here in https://github.com/decidim/decidim/issues/5635:
> As an admin I want to better configure if Metrics get shown **on a given Participatory Process**, and as an user I want to see more data on Activity. Also the terminology is not the same on backend and frontend and it's frustrating.

htmlboy commented:
@carolromero I've changed the demo and screenshot to the process page and added some fixes. Anyway, I wouldn't left align the text inside the stats blocks (we should remove the separators then, in order to keep alignment balanced, and the whole structure will lose some appeal). Thoughts?

carolromero commented:
@htmlboy I think now it's well balanced, we go with your updated design! Thanks

agustibr commented:
@htmlboy this branch was merged into #5688 😄 
shall I close it?

mrcasals commented:
@agustibr yes, let's close it then! 😄
